config/application_routes.js

```diff
@@ -28,8 +30,7 @@ routes.use("/trouble_ticket_hub", require("./routes/trouble_ticket_hub_routes"))
 routes.use("/privacy_profile", require("./routes/privacy_profile"))
 routes.use("/privacy_profile_type", require("./routes/privacy_profile_type"))
 routes.use("/privacy_agreement", require("./routes/privacy_agreement"))
-
-
+routes.use("/privacy_agreement", require("./routes/privacy_agreement"))
```

Comment:
Why is this line duplicated? Could be a weird merge but it does need to be fixed please.

Reply:
removed